pyserini/trectools/_base.py

```diff
@@ -99,6 +152,71 @@ def rescore(self, method: RescoreMethod, rrf_k: int = None, scale: float = None)
     def to_numpy(self) -> np.ndarray:
         return self.run_data.to_numpy(copy=True)
 
+    def remove_by_qrels(self, qrels: Qrels, copy=True):
```

Comment:
how about `discard_qrels` and `retain_qrels` to be consistent with https://github.com/castorini/anserini-tools/blob/master/scripts/filter_run_with_qrels.py ?